src/world.ts

```diff
@@ -179,16 +189,12 @@ class World {
         return new User(this.server, user).init();
     }
 
-    async updateEnvTerrain(db, env) {
-        let walled = '';
```

Comment:
This is no longer needed, as it is very slow and we have the `string.repeat(num)` function new in ES6.

Additionally, it can be cached on the module so it doesn't need to be computed every update. (see line 218)